### PR TITLE
Update the transportation information to the venue page

### PR DESCRIPTION
**Pull request by yuko29**
@YenPeiChen07 I updated the transportation information to the venue page.

### CODE REVIEW

**Comment by yuko29** [comment]
@YenPeiChen07 The conference pictures are added to the venue page.

File: content/2023/contents/venue.mdx [hunk 1]
```diff
@@ -1,7 +1,76 @@
...
+Navigate to [MIRC](https://eic2.web.nycu.edu.tw/)：[Google Map](https://maps.app.goo.gl/MttEZQraQ3usSbj47?g_st=ic)  
+
+**School Parking Guidelines**   
+All white parking spaces (with white lines) within the campus are available for parking. Only vehicles with valid parking permits for faculty and staff issued by the school in the current year are allowed to park in the blue parking spaces (with blue lines). Parking is prohibited on red lines, hatched yellow lines, and pedestrian walkways.  
+
+## By THSR
+
+After arriving at THSR Hsinchu station, there are two ways to get to NYCU Guangfu Campus.
+
+1. Take Bus No. 182 or the bus BL1 to the "Guo Gou" stop. Get off and walk to [MIRC](https://eic2.web.nycu.edu.tw/). Click to see [the reference route]((https://maps.app.goo.gl/GU7nbFYjc5bPLJPD8)).
```
Comment (YenPeiChen07) [inline content/2023/contents/venue.mdx:39]: The link of reference route here is incorrect (this is for train) and invalid.

File: content/2023/contents/venue.mdx [hunk 2]
```diff
@@ -1,7 +1,76 @@
...
+
+1. Take Bus No. 182 or the bus BL1 to the "Guo Gou" stop. Get off and walk to [MIRC](https://eic2.web.nycu.edu.tw/). Click to see [the reference route]((https://maps.app.goo.gl/GU7nbFYjc5bPLJPD8)).   
+2. Take a taxi to the [MIRC](https://eic2.web.nycu.edu.tw/). The taxi fare is approximately NT$275-NT$300.  
+
+## By Train
+
+After arriving at the Hsinchu Railway Station, you can transfer to a bus to reach the NYCU Guangfu Campus.  
+
+* Take Bus No. 2 directly to the "National Chiao Tung University Guangfu Campus" stop.  
+* Take Bus No. 182 or the Blue 1 zone bus to the "Gou-Gou" stop. Get off and walk to [MIRC](https://eic2.web.nycu.edu.tw/). Click to see [the reference route]((https://maps.app.goo.gl/GU7nbFYjc5bPLJPD8)).
```
Comment (YenPeiChen07) [inline content/2023/contents/venue.mdx:47]: The link of reference route here is also invalid.

**Review by YenPeiChen07 — CHANGES_REQUESTED** [review]
@yuko29 Thank you for updating the venue information. Before merging it, there a few things to correct. Feel free to let me know if you have any question.

File: content/2023/contents/venue.mdx (hunk 1 above)
Comment (yuko29) [reply content/2023/contents/venue.mdx]: Ok. I've corrected it on the latest commit.

**Comment by yuko29** [comment]
@YenPeiChen07 I've corrected the links.

**Review by YenPeiChen07 — APPROVED** [review]

**Pull request merged**